README.md

```diff
@@ -159,6 +159,18 @@ Image: librenms/librenms:latest
 * `DISPATCHER_NODE_ID`: Unique node ID for your dispatcher service
 * `DISPATCHER_ARGS`: Additional args to pass to the [dispatcher service](https://github.com/librenms/librenms/blob/master/librenms-service.py)
 
+### Logrotate
+
+> **Note**
+>
+> Logrotate could be enabled to avoid large log files.
+
+> **Warning**
+>
+> When activated, logs exceeding the specified retention period will be deleted.
+
+* `LOGROTATE_ENABLED`: Set to `true` to enable logrotate for this container (default `false`)
```

Comment:
Just add this after https://github.com/librenms/docker/blob/c14d7213777520059de2321f6753cbdc85283b98/README.md?plain=1#L134. No need a dedicated section for it.

Also it's not for the container but LibreNMS service imo:

```suggestion
* `LOGROTATE_ENABLED`: Set to `true` to enable logrotate for LibreNMS logs (default `false`)
```

Reply:
For consistency let's just name it:

```suggestion
* `LOGROTATE`: Set to `true` to enable logrotate for this container (default `false`)
```